code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -818,6 +821,8 @@
 				W.forceMove(null)
 
 				to_chat(usr, "<span class='notice'>You install the [W.name].</span>")
+				if(src.can_diagnose())
```

Comment:
It completely doesn't matter at all. I still don't know why people got so upset about it for a few days.